website/blog/2022-01-13-docusaurus-2021-recap/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Docusaurus 2021 Recap
+authors: [Josh-Cena]
+tags: [recap]
+image: ./img/thumbnail.png
```

Comment:
Some suggestion:

Make the text more readable

Add the 30k milestone on the card?

![image](https://user-images.githubusercontent.com/749374/150770880-556ded62-2a68-4468-9f2d-059984d68f63.png)